resources/views/platform/components/completion-progress-page/milestones/table-row.blade.php

```diff
@@ -0,0 +1,51 @@
+@props([
+    'milestone' => [],
+])
+
+<?php
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Str;
+
+$kindLabel = '';
+if ($milestone['which'] === 'most recent') {
+    $kindLabel = 'Latest';
+} else {
+    $formatter = new NumberFormatter(session('locale'), NumberFormatter::ORDINAL);
+    $kindLabel = $formatter->format($milestone['which']);
+}
+
+if ($milestone['kind'] === 'mastered') {
+    $kindLabel .= ' mastery';
+} elseif ($milestone['kind'] === 'completed') {
+    $kindLabel .= ' completion';
+} elseif ($milestone['kind'] === 'beaten-hardcore') {
+    $kindLabel .= ' game beaten';
+}elseif ($milestone['kind'] === 'beaten-softcore') {
+    $kindLabel .= ' game beaten*';
```

Comment:
This does not appear to show a tooltip on hover for me as we discussed
![image](https://github.com/RetroAchievements/RAWeb/assets/4047771/afe48234-8b1e-4ff0-9781-ecdd741617da)

Reply:
Fixed in https://github.com/RetroAchievements/RAWeb/pull/1833/commits/5521ad5f127246f292d0fa55406e04124c64e27f.